src/Microsoft.DotNet.Wpf/src/WindowsBase/WindowsBase.csproj

```diff
@@ -439,6 +439,14 @@
     </ProjectReference>
   </ItemGroup>
   
+  <ItemGroup>
+     <!--
+     Allow test projects access to internals (but only those that require it). Arcade will generate a file like
+     "WindowsBase.InternalsVisibleTo.cs" with the proper assembly name and PublicKey for us
+     -->
+     <InternalsVisibleTo Include="DrtPackagingApis" />
```

Comment:
I would recommend adding this to `OtherAssemblyAttrs.cs` directly, and introducing the assembly name into `BuildInfo` first under `shared\RefAssemblyAttrs.cs`. 

This allows us to maintain a single uniform system for tracking `InternalsVisibleTo` relationships. Having this encoding inside a `csproj` makes this harder to maintain, and potentially troubleshoot problems in the future. 

Any assemblies requested internals access to `WindowsBase` etc. should have its key fingerprint declared clearly in `RefAssebmlyAttrs.cs`. Having a separate section in there for test assemblies would be preferable.

Reply:
That seems like a lot more maintenance and overhead. Arcade generates this all for us based on the key you are using and all you have to is add this in your .csproj. We could go about removing all of that and moving to this model, but that would be out of scope for this PR.

Reply:
When we tried to remove this from `otherassemlyattrs.cs` early in .NET Core 3 and "clean up" this information, we broke API compat widely and spent several days chasing the mistakes and fixing problems in the build, then resurrecting this information from history.

Reply:
can you elaborate more on what happened? how does changing `InternalsVisibleTo` attributes break API compat?